### PR TITLE
Divyanshu | Test | PROD - Remove the dropdown of advance receipt and advance receipt adjustment from transaction type under Exempt and B2cs in GSTR 1 report

### PR DESCRIPTION
**Pull request by dvCodeWorld**
https://app.clickup.com/t/86cwvcymb

### CODE REVIEW

File: apps/web-giddh/src/app/gst/filing/tabs/overview/view-transactions/view-transactions.component.ts [hunk 1]
```diff
@@ -143,8 +143,13 @@ export class ViewTransactionsComponent implements OnInit, OnDestroy {
 
         this.activatedRoute.firstChild.queryParams.pipe(takeUntil(this.destroyed$)).subscribe(params => {
             this.selectedGstNumber = params.selectedGst;
-            this.filterParam.entityType = this.selectedGst === GstReport.Gstr2 ? params.entityType : '';
-            this.filterParam.type = params.type;
+            if ((params.entityType === 'registered-notes') || (params.entityType === 'unregistered-notes')) {
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/gst/filing/tabs/overview/view-transactions/view-transactions.component.ts]: check params in parent of if condition

File: apps/web-giddh/src/app/gst/filing/tabs/overview/view-transactions/view-transactions.component.ts [hunk 2]
```diff
@@ -105,8 +105,8 @@ export class ViewTransactionsComponent implements OnInit, OnDestroy {
             { label: this.commonLocaleData?.app_invoices, value: 'invoices' },
             { label: this.commonLocaleData?.app_credit_notes, value: 'credit-notes' },
             { label: this.commonLocaleData?.app_debit_notes, value: 'debit-notes' },
-            { label: this.localeData?.advance_receipt, value: 'advance-receipt' },
-            { label: this.localeData?.adjusted_advance_receipt, value: 'adjusted-advance-receipt' },
+            // { label: this.localeData?.advance_receipt, value: 'advance-receipt' },
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/gst/filing/tabs/overview/view-transactions/view-transactions.component.ts:108]: remove commented code

**Review by dilpreetsinghofficial — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/gst/filing/tabs/overview/view-transactions/view-transactions.component.ts (hunk 2 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/gst/filing/tabs/overview/view-transactions/view-transactions.component.ts:108]: Infuture we will use this option

File: apps/web-giddh/src/app/gst/filing/tabs/overview/view-transactions/view-transactions.component.ts (hunk 1 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/gst/filing/tabs/overview/view-transactions/view-transactions.component.ts]: Done

**Review by dilpreetsinghofficial — APPROVED** [review]

**Pull request merged**